run_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ ! -d "pefile-tests" ]; then
+    git clone git://github.com/viper-framework/pefile-tests.git
```

Comment:
Who operates this repository?
Why isn't this somewhere where the maintainer has control?